seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-7/src/test/resources/jdbc_mysql_source_and_sink.conf

```diff
@@ -27,7 +27,8 @@ source {
     connection_check_timeout_sec = 100
     user = "root"
     password = "Abc!@#135_seatunnel"
-    query = "select * from source;"
+    query = "select * from source"
+    table_path = seatunnel.source
   }
```

Comment:
why change this？

Reply:
If you do not add table_path, the Mysql database will fail to create a table failed

Reply:
Because the field information read by "Query SQL" is inaccurate

Reply:
So this change let some job cannot execute normally. Could you share the SQL of create table? Both of change before and change after.

Reply:
This unit test had a problem before this change, not this change caused

Reply:
how about create another pr to fix this problem? We should make sure this change not let any test case can not executed.